packages/core-dialog/core-dialog.js

```diff
@@ -102,7 +102,7 @@ function setOpen (dialog, open, opener = document.activeElement) {
 
   if (isUpdate) {
     const lastIndex = Number(lastFocus && lastFocus.getAttribute(OPENER_ATTR)) || 0
-    const topZIndex = Math.max(...queryAll('*').map(getZIndexOfElement))
+    const topZIndex = Math.max(...queryAll('*').map(getZIndexOfElement), 2000000000) // Avoid high z-indexes from extensions. techjunkie.com/maximum-z-index-value
```

Comment:
```suggestion
    const topZIndex = Math.min(Math.max(...queryAll('*').map(getZIndexOfElement)), 2000000000) // Avoid overflowing z-index. See techjunkie.com/maximum-z-index-value
```
Må være `Math.min` for å velge det laveste tallet - my bad - litt dårlig formulert issue :)